app/src/main/java/com/karumi/katasuperheroes/ui/view/BaseActivity.java

```diff
@@ -17,22 +17,27 @@
 package com.karumi.katasuperheroes.ui.view;
 
 import android.os.Bundle;
+import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
 import android.view.View;
 import com.karumi.katasuperheroes.R;
 import com.karumi.katasuperheroes.ui.presenter.Presenter;
+import butterknife.Bind;
+import butterknife.ButterKnife;
 
 public abstract class BaseActivity extends AppCompatActivity implements Presenter.View {
 
-  private Toolbar toolbar;
-  private View loadingView;
+  @Nullable @Bind(R.id.toolbar)
+  Toolbar toolbar;
```

Comment:
we use inline for annotations can you change it to:

@Nullable @Bind(R.id.toolbar) Toolbar toolbar;
@Nullable @Bind(R.id.progress_bar) View loadingView;